deployment/ansible/dist-example/env.yml

```diff
@@ -83,6 +83,8 @@ jophiel_userResetPassword_requestEmailTemplate_body: xxx
 jophiel_userResetPassword_resetEmailTemplate_subject: xxx
 jophiel_userResetPassword_resetEmailTemplate_body: xxx
 
+jophiel_superadmin_initial_password: xxx
```

Comment:
```suggestion
jophiel_superadmin_initialPassword: xxx
```

Reply:
OK

Reply:
Changed to `jophiel_superadminCreator_initialPassword` for consistency with actual config key